pom.xml

```diff
@@ -42,7 +47,7 @@
     </repositories>
 
     <properties>
-        <carbon.apimgt.version>6.7.206</carbon.apimgt.version>
+        <carbon.apimgt.version>9.0.2-SNAPSHOT</carbon.apimgt.version>
```

Comment:
shall we used a release version?

Reply:
changed in 09f0f2a0ec6b148f13a8092bf53055c9df346ef2